waypoint_x: [1,1,2,3,3]
waypoint_y: [1,3,4,3,1]
waypoint_th: [1.57079, 0.785398, -0.785398, -1.57079, -3.14159]